src/platform-plugin/platform-plugin.pro

```diff
@@ -1,7 +1,9 @@
 TARGET = qgtk
 
-CONFIG -= release
-CONFIG += debug
+!gtkplatform-release {
```

Comment:
Hm, I wonder if it's better to just remove this entirely (and pick up the default from the Qt build, and allow you to override it at qmake time)?

Reply:
I am easy; I just wanted to retain the same default behavior you peeps had in place for development, and allow packaging a release build